.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# Please see the documentation for all configuration options:
```

Comment:
Let's change the default branch to `develop` for the time being (since this config file is apparently picked up from the repo's default branch)